README.md

```diff
@@ -43,6 +43,8 @@ pip install .
 
 And `hokusai` should now be installed on your PATH.
 
+Ensure the environment variables `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and optionally `AWS_ACCOUNT_ID` are set in your shell.
+
```

Comment:
This isn't strictly required until the `init` step. Maybe move it there, and mention the flag alternatives (`--aws-account-id`, etc.)?